README.md

```diff
@@ -276,6 +276,15 @@ The above will not run StacktraceModelTest, as that is also matched by "test.exc
 
 Note that if UI-tests are supposed to be part of the filtered run the "uitests" profile needs to be specified as well. Otherwise the UI won't start up and so the tests fail.
 
+
+## Building using docker and docker-compose
+
+```
+docker-compose run -f docker/docker-compose.yml
```

Comment:
I think this command is not complete? It's needs a service to run as well?

Reply:
Thanks, good spot